test/e2e/issuer.ts

```diff
@@ -1,32 +1,51 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import fetch from 'node-fetch'; // using node-fetch to allow usage of client side certificate with a fetch-like API. This is not a standard webcrypto behaviour, therefore using native node fetch does not work.
+import fetch from 'node-fetch';
 import fs from 'node:fs';
 import https from 'node:https';
-
 import {
 	IssuerConfig,
 	MediaType,
 	PRIVATE_TOKEN_ISSUER_DIRECTORY,
 	TOKEN_TYPES,
 	TokenChallenge,
 	publicVerif,
+	arbitraryBatched,
 	util,
 } from '@cloudflare/privacypass-ts';
 
 const { BlindRSAMode, Client, Origin } = publicVerif;
+const { TokenRequest, Client: BatchedTokensClient, BatchedTokenResponse } = arbitraryBatched;
+import { type Token } from '@cloudflare/privacypass-ts';
+
+// ========== TYPES ==========
+
+export type IssuerConfiguration = {
+	publicKeyEnc: Uint8Array;
+	publicKey: CryptoKey;
+	url: string;
+};
 
 export interface MTLSConfiguration {
 	certPath: string;
 	keyPath: string;
 }
 
 export function base64UrlToUint8Array(base64Url: string): Uint8Array {
```

Comment:
worth just using https://github.com/cloudflare/privacypass-issuer/blob/main/src/utils/base64.ts instead of rewritting the tool